src/librbd/cache/pwl/rwl/WriteLog.cc

```diff
@@ -269,7 +267,7 @@ bool WriteLog<I>::initialize_pool(Context *on_finish, pwl::DeferredContexts &lat
                         this->m_pwl_pool_layout_name,
                         this->m_log_pool_size,
                         (S_IWUSR | S_IRUSR))) == NULL) {
-      lderr(cct) << "failed to create pool (" << this->m_log_pool_name << ")"
+      lderr(cct) << "Failed to create pool (" << this->m_log_pool_name << ")"
                  << pmemobj_errormsg() << dendl;
```

Comment:
I don't know if it was not in the initial version, or I just did not notice this. Anyway, the librbd convention seems to be alway use lowercase in debug/error messages. Why did you decided to change it here in other messages to start with uppercase?

Reply:
Sorry, I noticed that some places use uppercase and some places use lowercase. I unified them into uppercase. Now it has been unified into lowercase. Thanks, I learned.